compile.sh

```diff
@@ -1,20 +1,23 @@
 #!/bin/sh -x -e
 
-# Note that the build system needs to be Apple Silicon with Rosetta 2 enabled.
+# Note that the build system needs to be BigSur or newer, Intel or Apple Silicon
```

Comment:
Can you please update this to specify macOS 11.3 rather than Big Sur?  There's a fix in 11.3 that fixes #50.